CHANGELOG.md

```diff
@@ -5,9 +5,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
-## [v1.0.0](https://github.com/puppetlabs/puppetlabs-node_manager/tree/v1.0.0) - 2024-06-07
+## [v1.0.1](https://github.com/puppetlabs/puppetlabs-node_manager/tree/v1.0.1) - 2024-07-05
```

Comment:
ah I forgot that the changelog is based on git tags. so It's really good that we've updated the README.md that expains the missing tag.